public/app/core/components/manage_dashboards/manage_dashboards.html

```diff
@@ -114,14 +114,15 @@
 </div>
 
 <div ng-if="ctrl.canSave && ctrl.folderId && !ctrl.hasFilters && ctrl.sections.length === 0">
-  <empty-list-cta model="{
-    title: 'This folder doesn\'t have any dashboards yet',
-    buttonIcon: 'gicon gicon-dashboard-new',
-    buttonLink: 'dashboard/new?folderId={{ctrl.folderId}}',
-    buttonTitle: 'Create Dashboard',
-    proTip: 'Add/move dashboards to your folder at ->',
-    proTipLink: 'dashboards',
-    proTipLinkTitle: 'Manage dashboards',
-    proTipTarget: ''
-  }" />
+  <!--angular is evaluating these expressions and they need to be references instead of string literals directly-->
```

Comment:
Ah sorry, I meant comment as in just Github comment for this particular PR (like this one) :). The information is definitely useful but on the other hand repeating the same comment multiple times is something I like to avoid (easy to get out of sync, same as duplicated code). So as there is no sensible single location to put this where it would be useful and easy to discover I would just leave out the comments.

Reply:
I've removed comments, as they are not needed anymore.